clang/lib/Driver/Driver.cpp

```diff
@@ -4798,6 +4798,11 @@ class OffloadingActionBuilder final {
     /// Running list of SYCL actions specific for device linking.
     ActionList SYCLLinkBinaryList;
 
+    /// List of SYCL Final Device binaries that should be unbundled as a final
+    /// device binary and not further processed.
+    SmallVector<std::pair<Action *, SmallVector<std::string, 4>>>
```

Comment:
Is the number of elements parameter for `SmallVector` class intentionally left out? All other usage of `SmallVector `with `std::pair` as a parameter, have the number of elements mentioned.